src/ts/src/NavBar.module.scss

```diff
@@ -2,82 +2,99 @@
 @import "src/common/styles/spacing";
 @import "src/common/styles/typography";
 
+@mixin item-common {
+  height: 100%;
+  font-family: $font-family-primary;
+  color: $white;
```

Comment:
why is white sometimes a variable and sometimes a constant?

Reply:
I think only the logo one is constant `white`, the rest should be variable `$white`. And the reason is that the logo white shouldn't depend on the variable white, in case the design wants to change the white variable to a different shade of white, and we want the logo to stay the same!

Reply:
Should we possibly do `$aspen-background` for the `$white` variable or `$aspen-white`?  Not sure what the best practices are, but I kind of suspect that just `$white` will cause confusion in the long run.

Reply:
Oh yes for best practice, IDseq's design system has defined color variables and other semantic variable names already. So all variables we use in our code base follow their design system 😄 !

https://www.figma.com/file/EaRifXLFs54XTjO1Mlszkg/Science-Design-System-Reference?node-id=0%3A3498

So `$white` in Aspen maps to the design system one, like so:

<img width="748" alt="Screen Shot 2021-04-16 at 11 37 17 AM" src="https://user-images.githubusercontent.com/6309723/115069253-2b1dce00-9ea8-11eb-82fd-a2e09a2274cb.png">